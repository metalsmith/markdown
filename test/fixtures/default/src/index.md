---
not_a_string: false
---
# A Markdown Post

With some "amazing", _riveting_, **coooonnnntent**.