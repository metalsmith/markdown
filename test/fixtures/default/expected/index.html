<h1 id="a-markdown-post">A Markdown Post</h1>
<p>With some &quot;amazing&quot;, <em>riveting</em>, <strong>coooonnnntent</strong>.</p>
